src/tests-event-replay/import-export-tests.ts

```diff
@@ -99,3 +106,81 @@ describe('import/export tests', () => {
     await db?.close();
   });
 });
+
+describe('Events table', () => {
```

Comment:
Test is great. Since you're declaring a new `describe` block, you should send this to a new file inside `src/tests-event-replay`. I'd suggest `src/tests-event-replay/raw-event-request-tests.ts`